omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -241,6 +241,11 @@ Database creation
 ^^^^^^^^^^^^^^^^^
 
 Start the PostgreSQL server.
+If you wish to use ``brew services`` to start postgresql, you will first need to 
+install it:
+
+::
+    $ brew tap gapple/services
```

Comment:
This isn't rendered correctly on staging - you need a blank line before the '$ brew…' content